angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome-modal-template.html

```diff
@@ -0,0 +1,12 @@
+<div class="welcome-popup center">
+  <h1 ng-bind-html="welcome.title"></h1>
+  <img src="{{ welcome.img }}" class="hidden-xs">
+	<div class="sr-only" tabindex="0" id="welcomeContent" aria-labelledby="welcomeContent">
```

Comment:
Cool.  I don't want to lose sight of the progress we're making: having this accessibility text at all is better than not having it, and is definitely progress, and is to be appreciated and commended. :cake: 

Looking around, I think this may be an opportunity to dig into the distinction between labels, which give non-visual users pithy handles to understand the objects on the page, and descriptions, which give non-visual users an alternative way to understand the content of the objects on the page.  Our very own [DoIT accessibility guidelines](http://www.doit.wisc.edu/outside-cms/accessibility/online-course/standards/images.htm) seem to have some relevant advice.